server/serverAPI/controller/InventoryController.ts

```diff
@@ -61,11 +114,28 @@ export default class InventoryController extends BaseUserController {
     getAll = async (req: Request, res: Response) => {
         let parameters = new Map<string, any>([["username", req.serverUser.username]]);
 
+        let isReverse = req.query.isReverse === 'true' ? true : false;
+
         try {
             let user = await this.requestGet(parameters, res)
-            return this.send(ResponseCodes.OK, res, user.inventory);
-        } catch (response) {
-            return response;
+
+            let responseData: any = user.inventory;
```

Comment:
It's any because it's a response data... Since you are asking me to return different objects for different sorting methods, there is no way to create a single class which would describe all of them... It could be done through interfaces but I'm too tired to create 3 more interfaces with the base interface each of which would describe sortings and stuff.